x-pack/plugins/security_solution/common/endpoint/schema/trusted_apps.ts

```diff
@@ -30,7 +30,10 @@ export const GetTrustedAppsRequestSchema = {
 };
 
 const ConditionEntryTypeSchema = schema.literal('match');
-const ConditionEntryOperatorSchema = schema.literal('included');
+const ConditionEntryOperatorSchema = schema.oneOf([
```

Comment:
I would love to be able to do a more specific schema validation where we check that operator has two possible values only when we select **path** field. I could use some help with that.

Reply:
I think a [conditional](https://github.com/elastic/kibana/tree/4584a8b570402aa07832cf3e5b520e5d2cfa7166/packages/kbn-config-schema#schemaconditional) is what you are looking for

Reply:
Yes 👍